identity/identity.proto

```diff
@@ -200,6 +200,27 @@ message Capability {
     Type type = 1;
   }
 
+  // EncryptionKeyRotation contains the features of the EncryptionKeyRotation
+  // operation that the CSI-driver supports.
+  message EncryptionKeyRotation {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the
+      // EncryptionKeyRotation operation in the current mode.
+      // The CSI-Addons CO plugin will most likely ignore this node
+      // for the EncryptionKeyRotation operation.
+      UNKNOWN = 0;
+      // ENCRYPTIONKEYROTATION indicates that the CSI-driver provides
+      // RPCs for an EncryptionKeyRotation operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require a volume to be staged/attached to
+      // the node.
+      ENCRYPTIONKEYROTATION = 1;
```

Comment:
maybe call this `NODE_ENCRYPTIONKEYROTATION` so that `CONTROLLER_ENCRYPTIONKEYROTATION` can be added later if it would be useful?

Reply:
Acc. to discussions above, leaving this as is.